external/stdk/tizenrt-wrapper/bsp/iot_bsp_wifi_common.c

```diff
@@ -109,7 +110,7 @@ static void _obtain_time(void)
 	}
 
 	// wait for time to be set
-	while (timeinfo.tm_year < (2016 - 1900) && ++retry < retry_count) {
+	while (timeinfo.tm_year < byear && ++retry < retry_count) {
```

Comment:
The time has been reset to year 1970  when starting NTP (in function _initialize_ntp), so we can check condition `timeinfo.tm_year > TM_YEAR_BASE` to judge if the time is synchronized or not.

Reply:
Maybe the time should be saved before reset, and restored in case of NTP syncing failed...
Please consider this.

Reply:
time is reset to 1970, and TM_BASE_YEAR is 1900, here let's check tm_year <= (EPOCH_YEAR - TM_YEAR_BASE) for invalid time and delay to wait for system time set